docs/feature/connectivity/index.md

```diff
@@ -1,85 +1,608 @@
+            properties.put("user", "crate");
+            properties.put("password", "");
+            properties.put("ssl", false);
+            Connection conn = DriverManager.getConnection(
+                "jdbc:crate://localhost:5432/",
+                // "jdbc:postgresql://localhost:5432/"
+                properties
+            );
+
+            Statement statement = conn.createStatement();
+            ResultSet resultSet = statement.executeQuery("SELECT * FROM sys.summits LIMIT 3;");
+            resultSet.next();
+            String mountain = resultSet.getString("mountain");
+
+            System.out.println(mountain);
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+}
+```
+:::::
+
+:::::{tab-item} JavaScript / node-postgres
+For more information see [node-postgres] documentation.
+```shell
+npm install pg
+```
+```javascript
+const { Client } = require("pg");
+
+const cratedbClient = new Client({
+  host: "localhost",
+  port: 5432,
+  user: "crate",
+  password: "",
+  ssl: false,
+});
+
+(async () => {
+  await cratedbClient.connect();
+  const result = await cratedbClient.query("SELECT * FROM sys.summits LIMIT 3;");
+  console.log(result.rows);
+})();
+```
+:::::
+
+:::::{tab-item} JavaScript / node-crate
+`node-crate` is an independent Node.js driver implementation for
+CrateDB, using the `/_sql` HTTP API endpoint.
+For more information see [node-crate] documentation.
+```shell
+npm install node-crate
+```
+```javascript
+const crate = require("node-crate");
+
+crate.connect(`https://crate:${encodeURIComponent("")}@localhost:4200`);
+
+(async () => {
+  const result = await crate.execute("SELECT name FROM sys.cluster");
+  console.log(result.rows[0]);
+})();
+```
+:::::
+
+:::::{tab-item} Python / DB API
+For more information see [CrateDB Python] documentation.
+```shell
+pip install --upgrade crate
+```
+```python
+import crate.client
+from pprint import pprint
+
+connection = crate.client.connect("http://localhost:4200")
+cursor = connection.cursor()
+cursor.execute("SELECT * FROM sys.summits LIMIT 3;")
+results = cursor.fetchall()
+pprint(results)
+cursor.close()
+connection.close()
+```
+:::::
+
+:::::{tab-item} Python / SQLAlchemy
+For more information see [CrateDB SQLAlchemy] documentation.
+```shell
+pip install --upgrade sqlalchemy-cratedb
+```
+```python
+import sqlalchemy as sa
+from pprint import pprint
+
+engine = sa.create_engine("crate://localhost:4200/doc", isolation_level="AUTOCOMMIT", echo=True)
+
+with engine.connect() as connection:
+    cursor = connection.execute(sa.text("SELECT * FROM sys.summits LIMIT 3;"))
+    pprint(cursor.fetchall())
+```
+:::::
+
+::::::
+
 
-:::{rubric} Community Tutorials
+:::::{dropdown} PHP
+
+:::{rubric} HTTP
+:::
+
+::::{dropdown} PDO_CRATEDB
+The PHP Data Objects (PDO) is a standard PHP extension that defines a common
+interface for accessing databases in PHP.
+For more information see [CrateDB PDO] documentation.
```

Comment:
That's the "dropdown" variant, already using **nested dropdowns**, immediately applicable with PHP and Python for example, as there are many driver variants for them, across two families (HTTP vs. PostgreSQL), which stretches all axes of the system needed to represent those.

We are biased to use this variant, and drop the "tabbed" variant again. wdyt?

Reply:
The main question to you here is if we should continue with the dropdown variant. Thanks!

Reply:
If we go with the "tabbed" variant, I think we should pick 3-4 items only. So if we want to have all items, the "dropdown" variant might be better.

Reply:
I tried to present both variants without many initial plans, and just related to a recent comment from your pen that you like "tabbed" approaches of information conveyance.

What do _you_ think how we should proceed on this particular matter?

Reply:
Thanks for your suggestions. I've diverted corresponding updates into a separate patch that needs more love, in order to get this one merged.

- GH-138